lms/envs/common.py

```diff
@@ -2268,6 +2268,7 @@
 STATIC_TEMPLATE_VIEW_DEFAULT_FILE_EXTENSION = 'html'
 
 SUPPORT_SITE_LINK = ''
+ACTIVATION_EMAIL_SUPPORT_LINK = ''
```

Comment:
Can you default to the existing support link here?

https://support.edx.org/hc/en-us/articles/227340127-Why-haven-t-I-received-my-activation-email-